### DISCUSSION TITLE
E: Unable to locate package wsl

### DISCUSSION THREAD

**Discussion by veltgaist** [microsoft/WSL#6983] [Q&A] [answered]
### Windows Build Number

Microsoft Windows [Version 10.0.19041.985]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

Invalid command line option: --status

### Distro Version

Ubuntu 20.04 LTS

### Repro Steps

sudo apt install wsl
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package wsl

### Expected Behavior

Install WSL, I did update and upgrade

### Actual Behavior

WSL not working properly

**Comment by SheepReaper** [comment, marked as answer]
A. did you `sudo apt update` first?
B. Are you looking for the right thing? _wsl_ the *apt package* is WSMan Shell not Windows Subsystem for Linux

Apt is not managed by Microsoft. If you're looking for help with WSMan Shell, this is not that.
